lib/PRC/Schema/Result/User.pm

```diff
@@ -57,6 +57,10 @@ __PACKAGE__->add_columns(
   { data_type => "integer", default_value => 0, is_nullable => 0 },
   "assignee_level",
   { data_type => "integer", default_value => 0, is_nullable => 0 },
+  "share_email_user_assignment",
```

Comment:
Let's go with names that reflect what it does clearly. Something like,

    is_sharing_email_with_assignment_repo_owners
    is_sharing_email_with_assignees